components/org.wso2.carbon.identity.oauth.ui/src/main/resources/org/wso2/carbon/identity/oauth/ui/i18n/Resources.properties

```diff
@@ -84,6 +84,9 @@ error.while.revoking.authorization=Error while revoking authorization for {0}
 revocation.success=Authorization successfully revoked for {0}
 next=Next
 prev=prev
+user.access.token.expiry.time=User Access Token Expiry Time in Milliseconds
```

Comment:
Why does this have to defined in Milliseconds?

Reply:
If seconds needed we can put that as well as User Access Token Expiry Time in seconds

Reply:
IMO seconds is more natural to a user than milliseconds. WDYT?